msfrpc

```diff
@@ -41,7 +41,7 @@ opts = {
 }
 
 # Parse command line arguments.
-arguments.parse(ARGV) { |opt, idx, val|
+arguments.parse(ARGV) do |opt, idx, val|
```

Comment:
Normally I'm a huge fan of idiom, but alas, the prior true/truthy business is solidly anti-idiomatic, so now I'm gunshy.